docs/reference/migration/migrate_3_0.asciidoc

```diff
@@ -807,6 +807,17 @@ values `off`, `false`, `0`, and `no` for disabling a scripting mode.
 The variants `on`, `1`, and `yes ` for enabling and `off`, `0`,
 and `no` for disabling are no longer supported.
 
+==== Groovy dependencies
```

Comment:
Should we add here `[[breaking_30_groovy_deps]]`? So we will have a link to this section?